notebooks/name-entity-extraction.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
<p>In the conclusion:</p><p>NIT: "showed us how how" -&gt; "showed how easy..."</p><blockquote>showing us how easy and powerful it is when being used as a back end for an API.</blockquote><p>NIT: "it is when being used" -&gt; "when used"</p><p> </p><p>Sentence 2: Be more specific about which purposes and tasks, and add a callout to other tasks (with a link to the HUB)</p><p> </p><p>Sentence 3: would also benefit from being more specific, maybe mention real-time interaction for a person? Say in what way it is powerful (speed) and easy (Numbers of lines of code)? Maybe mention that this approach would also support other applications, like translation? sentiment analysis?</p><p> </p><p>Conclusion needs a next step/opening: go and see another related notebook? Maybe the one on finetuning for token classification?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/optimum-graphcore-fork/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>